### ISSUE TITLE
Aggregate Accuracy Score calculation

### ISSUE THREAD

**Issue by curryqka** [XiandaGuo/Drive-MLLM#5]
Hi，@XiandaGuo
# Issue: Clarification on AccS (Aggregate Accuracy Score) Calculation and Example

**Description:**

We are interested in understanding the calculation method for the AccS (Aggregate Accuracy Score) metric mentioned in the paper. Could you please provide a detailed explanation of how AccS is computed and, if possible, an example using the provided experimental data?

**Background:**

In the paper, AccS is mentioned as a key metric for evaluating performance. However, the exact formula or methodology for calculating AccS is not clearly specified. To better understand and reproduce the results, we need a clear definition and example of how AccS is derived from the experimental data.

**Request:**

1. **Definition of AccS:** Could you provide the exact formula or steps used to calculate AccS?
2. **Example Calculation:** Using the provided CSV data, could you walk through an example of how AccS is calculated? For instance, how would the values from the CSV (e.g., `xy_score`, `box_score`, `effective_ratio`, etc.) contribute to the final AccS?

**Provided Data:**

Here is the experimental data from the CSV file:

| metric               | value                                      |
|----------------------|--------------------------------------------|
| xy_score             | 28.99                                      |
| 0_effective_ratio_str| 823/880/880/880                            |
| 0_effective_ratio    | 93.52272727272727                          |
| box_score            | 3.65                                       |
| 1_effective_ratio_str| 879/879/880/880                            |
| 1_effective_ratio    | 99.88636363636364                          |
| left_score           | 83.02                                      |
| 2_effective_ratio_str| 159/191/191/191                            |
| 2_effective_ratio    | 83.24607329842932                          |
| right_score          | 64.17                                      |
| 3_effective_ratio_str| 187/191/191/191                            |
| 3_effective_ratio    | 97.90575916230367                          |
| front_score          | 51.83                                      |
| 4_effective_ratio_str| 191/191/191/191                            |
| 4_effective_ratio    | 100.0                                      |
| behind_score         | 51.83                                      |
| 5_effective_ratio_str| 191/191/191/191                            |
| 5_effective_ratio    | 100.0                                      |
| dA_score             | 46.61                                      |
| 6_effective_ratio_str| 72/692/880/880                             |
| 6_effective_ratio    | 8.181818181818182                          |
| dz_score             | 50.19                                      |
| 7_effective_ratio_str| 859/875/880/880                            |
| 7_effective_ratio    | 97.61363636363637                          |
| dAB_score            | 34.02                                      |
| 8_effective_ratio_str| 2/82/191/191                               |
| 8_effective_ratio    | 1.0471204188481675                         |
| dx_score             | 43.8                                       |
| 9_effective_ratio_str| 22/95/191/191                              |
| 9_effective_ratio    | 11.518324607329843                         |

**Expected Outcome:**

- A clear explanation of how AccS is calculated.
- An example using the provided data to illustrate the calculation process.
- If possible, the final AccS value derived from the provided data.

**Thank you for your assistance!**

**Comment by bonbon-rj** [comment]
Thank you for your interest and feedback!
The metric AccS is computed as the average of eight evaluation scores:
AccS = (xy_score + box_score + (left_score + right_score) / 2 + (front_score + behind_score) / 2 + dA_score + dz_score + dAB_score + dx_score) / 8

**Comment by curryqka** [comment]
> Thank you for your interest and feedback! The metric AccS is computed as the average of eight evaluation scores: AccS = (xy_score + box_score + (left_score + right_score) / 2 + (front_score + behind_score) / 2 + dA_score + dz_score + dAB_score + dx_score) / 8

@bonbon-rj Hi, Regarding the AccS, if you previously thought it was calculated as the average of these eight evaluation scores, it seems there might be some confusion. Based on your clarification, AccS is indeed the average of these scores. If there is another metric called "Overall Score"(the last column in the Tab.3 in your paper) that is different from AccS, could you please provide additional details on how it is calculated?

**Comment by bonbon-rj** [comment]
Thank you for your question. Regarding your concern, the calculation of the final score is as follows:
```text
Score = (
xy_score*0_effective_ratio/100 + 
box_score*1_effective_ratio/100 + 
(left_score*2_effective_ratio/100 + right_score*3_effective_ratio/100) / 2 + 
(front_score*4_effective_ratio/100 + behind_score*5_effective_ratio/100) / 2 + 
dA_score*6_effective_ratio/100 + 
dz_score*7_effective_ratio/100 + 
dAB_score*8_effective_ratio/100 + 
dx_score*9_effective_ratio/100
) / 8
```

**Comment by curryqka** [comment]
> Thank you for your question. Regarding your concern, the calculation of the final score is as follows:
> 
> ```
> Score = (
> xy_score*0_effective_ratio/100 + 
> box_score*1_effective_ratio/100 + 
> (left_score*2_effective_ratio/100 + right_score*3_effective_ratio/100) / 2 + 
> (front_score*4_effective_ratio/100 + behind_score*5_effective_ratio/100) / 2 + 
> dA_score*6_effective_ratio/100 + 
> dz_score*7_effective_ratio/100 + 
> dAB_score*8_effective_ratio/100 + 
> dx_score*9_effective_ratio/100
> ) / 8
> ```

Thanks for your clarification!

**Comment by curryqka** [comment]
> Thank you for your question. Regarding your concern, the calculation of the final score is as follows:
> 
> ```
> Score = (
> xy_score*0_effective_ratio/100 + 
> box_score*1_effective_ratio/100 + 
> (left_score*2_effective_ratio/100 + right_score*3_effective_ratio/100) / 2 + 
> (front_score*4_effective_ratio/100 + behind_score*5_effective_ratio/100) / 2 + 
> dA_score*6_effective_ratio/100 + 
> dz_score*7_effective_ratio/100 + 
> dAB_score*8_effective_ratio/100 + 
> dx_score*9_effective_ratio/100
> ) / 8
> ```

Hi @bonbon-rj , I hope this message finds you well. I am currently working with your benchmark and have a couple of questions regarding its scope and future developments.
Benchmark Camera Views:
I noticed that the benchmark currently seems to focus primarily on front-view images. I am curious to know if this is the case, or if the benchmark also includes images from other camera angles (e.g., front-left views, back views, etc. in nuScenes). In my opinion, utilizing all six possible camera views could significantly enhance the robustness and versatility of the benchmark. 

I am also interested in knowing if there are any plans to release the training code for this benchmark in the future.

**Comment by bonbon-rj** [comment]
Thank you very much for your interest! The evaluation of multi-view images and the model training are currently in progress. Please stay tuned!
